doc/user/content/releases/v0.36.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Materialize v0.36"
+date: 2022-12-14
+released: true
+---
+
+## v0.36.0
+
+* Add `mz_internal.mz_sink_status` and `mz_internal.mz_sink_status_history`
+  to the system catalog. These objects respectively expose the current and
+  historical state for each sink in the system, including potential error
+  messages and additional metadata helpful for debugging.
+
+* Add `mz_internal.mz_cluster_replica_sizes` to the system catalog. This table
+  provides a mapping of logical sizes (e.g. `xlarge`) to the number of
+  processes, as well as CPU and memory allocations for each process. To monitor
+  the resource utilization for all extant cluster replicas as a % of the total
+  allocation, you can now use:
+
+  ```sql
+  SELECT
+    r.id AS replica_id,
+    m.process_id,
+    m.cpu_nano_cores / s.cpu_nano_cores * 100 AS cpu_percent,
+    m.memory_bytes / s.memory_bytes * 100 AS memory_percent
+  FROM mz_cluster_replicas AS r
+  JOIN mz_internal.mz_cluster_replica_sizes AS s ON r.size = s.size
+  JOIN mz_internal.mz_cluster_replica_metrics AS m ON m.replica_id = r.id;
+  ```
```

Comment:
Should we add a warning to these `mz_internal` objects that they may change at any time?

Reply:
It's in the documentation, so definitely doesn't hurt.